src/lib/storage/proposal/settings.rb

```diff
@@ -82,6 +84,7 @@ def initialize
           # Not yet in control.xml
           @home_min_size                 = DiskSize.GiB(10)
           @home_max_size                 = DiskSize.unlimited
+          @useful_free_space_min_size    = DiskSize.MiB(30)
```

Comment:
Being able to easily share it is a good argument. But it should be clear that this may never become a parameter for PMs to configure; this is strictly technical.

In the not-so-far future this might even become a value that will be calculated from our own partition alignment logic: It will be something like one sector less than the alignment size.

But I can live with this for the time being.